tv_apps/smart-home/js/connection_manager.js

```diff
@@ -0,0 +1,33 @@
+'use strict';
+
+(function(exports) {
+  var ConnectionManager = function() {};
+
+  ConnectionManager.prototype = evt({
+    init: function cm_init() {
+      window.addEventListener('iac-appdeck-channel', this);
+    },
+
+    uninit: function cm_uninit() {
+      window.removeEventListener('iac-appdeck-channel', this);
```

Comment:
I'm not sure if it's necessary to wrap IAC into `ConnectionManager` since we've already got `iac_handler.js`... maybe we can make a short discuss tomorrow.